ddp-workspace/projects/ddp-dsm-ui/src/app/sequencing-order/sequencing-order.component.ts

```diff
@@ -159,4 +161,23 @@ export class SequencingOrderComponent {
   closeModal(): void {
     this.modal.hide();
   }
+
+  public canNotSubmitOrder(): boolean{
+    return !this.role.allowedToDoOrderSequencing();
+  }
+
+  canEditCollectionDate(sample : SequencingOrder ) {
+  if (sample.sampleType === 'Normal') {
```

Comment:
Do you want a return type?
Is the indenting a bit off, or is this just a GIthub view thing?

Reply:
+1

Reply:
fixed, thank you :)